doc/lettable-operators.md

```diff
@@ -95,11 +95,13 @@ const takeEveryNth = (n: number) => <T>(source: Observable<T>) =>
     })
   });
 
+const takeEveryNthSimple = (n: number) => <T>(source: Observable<T>) =>
+  source.pipe(filter((value, index) => index % n === 0 ))
 
 interval(1000).pipe(
   takeEveryNth(2),
   map(x => x + x),
-  takeEveryNth(3),
+  takeEveryNthWithPipe(3),
```

Comment:
Did you mean to make this `takeEveryNthSimple`

Reply:
yes, I decided to rename it to Simple but forgot this place :)